### DISCUSSION TITLE
对参考文献格式的疑问

### DISCUSSION THREAD

**Discussion by tengxiao24** [sikouhjw/gdutthesis#30] [Q&A] [answered]
作者您好，我用gdutthesis模板生成的参考文献 作者名称全部都是大写，学校给的模板我看了一下是首字母大写。不知这里是否存在问题。

**Comment by sikouhjw** [comment, marked as answer]
1. 《撰写规范》[^1]规定了
   > 参考文献格式参照国家标准“GB7714-2005 文后参考文献著录规则”
2. 参考文献最新标准为《GB/T 7714-2015》[^2]
3. 《撰写规范》的规定很明显是过时且无意义的
4. 《GB/T 7714-2015》规定了
   > 6.3 个人著者，其姓全部著录，字母全大写，名可缩写为首字母（见8.1.1）；如用首字母无法识别该人时，则用全名。
5. 你可以不使用全大写，请根据你选择的参考文献宏包去阅读 `texdoc gbt7714`[^3] 或 `texdoc biblatex-gb7714-2015`[^4]
6. 你还可以不用我设置的功能，自行用符合《GB/T 7714-2005》[^5]标准的 bst 文件，这是你的自由

[^1]: https://yjs.gdut.edu.cn/system/_content/download.jsp?urltype=news.DownloadAttachUrl&owner=1533825352&wbfileid=3799073
[^2]: http://openstd.samr.gov.cn/bzgk/gb/newGbInfo?hcno=7FA63E9BBA56E60471AEDAEBDE44B14C
[^3]: https://www.ctan.org/pkg/gbt7714
[^4]: https://www.ctan.org/pkg/biblatex-gb7714-2015
[^5]: http://openstd.samr.gov.cn/bzgk/gb/newGbInfo?hcno=15A0CA77357293622B8C6E9CDD19C5E5

**Comment by sikouhjw** [comment]
`texdoc gbt7714` 提供了 `gbt7714-2005-numerical.bst`，直接调用即可，这是使用者的自由。

PS：本科的规定更加离谱，规定是《GB/T 7714-1987》[^1]。

[^1]: http://openstd.samr.gov.cn/bzgk/gb/newGbInfo?hcno=ED1A66518439822B36403D707D55F92A

**Comment by sikouhjw** [comment]
> PS：本科的规定更加离谱，规定是《GB/T 7714-1987》

在《2022届毕业设计（论文）手册》[^1]中，参考文献规范已修改为《GB/T 7714-2015》
> 参考文献的著录应按中华人民共和国国家标准 GB/T 7714—2015 《信息与文献参考文献著录规则》执行。

[^1]: https://jwc.gdut.edu.cn/system/_content/download.jsp?urltype=news.DownloadAttachUrl&owner=1687414050&wbfileid=7575173
